### ISSUE TITLE
Acessibility

### ISSUE THREAD

**Issue by Kmoran1512** [lwbecker/Chord-Builder#1]
First let me say the game play is really fun, I found my self playing the game for a long time, but when trying to play with a screen reader I was unable to have it read the directions. I don't know if you've tried it bu settings the tabindex property of the heading and paragraph should fix the problem. Also as far as game play, you might what to play the correct chord the first time, so that the user has something to look for if not, as a person who can't read music and doesn't know what a chord looks like on paper, I don't know exactly what my goal is except to make the chord in the game sound major. Also if someone is blind they will be doing the same thing since they can't see the sheet music. (The screen reader I am using is ChromeVox)

**Comment by kirstenfreeman** [comment]
Thanks so much for your feedback! That's really helpful - we'll look into that and definitely fix the accessibility issue first.

**Comment by Kmoran1512** [comment]
Just went back and played it, it looks great and everything works well! Really great game.
